### ISSUE TITLE
Error connecting to console. On Steam deck

### ISSUE THREAD

**Issue by trebs24** [TheAssassin/AppImageLauncher#604]
### Pre-submit checks

- [X] I [checked for similar issues](https://github.com/TheAssassin/AppImageLauncher/issues?q=) beforehand, but could not find any, not even closed ones. I could not add my bug report to any existing issue.
- [X] I am going to take the time to to fill in all the required details. I know that the bug report will be dismissed otherwise.

### Describe the bug

Previously had beta 2. Updated to latest since 2 was no longer working
Upon installing, when trying to connect to console, I get this error after about 5 seconds. Closing the error takes you back to "We are getting your stream ready" indefinitely.

Here is the error message:
Stream error result: Failed
Details: [XccsClientHttpError] Xccs: Timeout : The request failed to complete in the allowed 00:00:05 timeout period: State 
WaitingForServerToRegister

Happens every time you try to connect to the console. For what it's worth, I can launch any game from cloud using Greenlight, but this error occurs when trying to connect directly to the console by using the "Start stream" option

### Expected behavior

Previously I did not have this error when connecting to the console through greenlight.

### Steps to reproduce the issue

Start Greenlight, Select console, hit "start stream"

### Distribution and desktop environment

On Steam Deck

### Installed AppImageLauncher version

2.2.0

### List of AppImages you tried

2.2.0

**Comment by TheAssassin** [comment]
I think you wanted to report an issue with Valve, not me.

**Comment by Thaunier** [comment]
Using the Linux build via a SteamDeck versus any other hardware running Linux shouldn't matter too much if the software is failing to connect with Xbox's API, no? I would hope it shouldn't matter if someone was using a SteamDeck, therefore routing them to Valve wouldn't be helpful if the issue was with the Linux build instead.

**Comment by TheAssassin** [comment]
Indeed. But it's a regular pattern that because some software comes as an AppImage, people create bug reports with Team AppImage and the community.
